spdm_dump/spdm/spdm_dump_spdm.c

```diff
@@ -2697,6 +2747,15 @@ void dump_spdm_encapsulated_response_ack(const void *buffer, size_t buffer_size)
             printf("(ReqSlotID=0x%02x) ",
                    *((uint8_t *)buffer + header_size));
```

Comment:
I recommend to define local variable req_slot_id, then `red_slot_id = *((uint8_t *)buffer + header_size`